ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -465,6 +466,8 @@ spec:
         name: content-hash
       runAfter:
         - apply-test-waivers
+      when:
```

Comment:
I am not sure if this is the expected behavior for new FBC-supported operator pipelines but here is the tested pipeline run: https://console-openshift-console.apps.pipelines-stage.0ce8.p1.openshiftapps.com/k8s/ns/haripate-playground/tekton.dev~v1~PipelineRun/operator-hosted-pipeline-runvf5rh

The test PR: https://github.com/redhat-openshift-ecosystem/community-operators-pipeline-preprod/pull/266

If this is not correct behavior then I would suggest having the If condition in a script of each affected tekton task rather than the when condition.